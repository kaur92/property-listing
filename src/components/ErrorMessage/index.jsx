import React from "react";
import "./ErrorMessage.css";

const ErrorMessage = () => (
  <div className="error-page">
    <h1>Oops!!</h1>
    <h3>Something went wrong please try again.</h3>
  </div>
);

export default ErrorMessage;
