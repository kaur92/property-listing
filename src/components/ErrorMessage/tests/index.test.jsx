import renderer from "react-test-renderer";
import ErrorMessage from "../index";

describe("<ErrorMessage /> tests", () => {
  it("renders correctly", () => {
    const cardComponent = renderer.create(<ErrorMessage />).toJSON();
    expect(cardComponent).toMatchSnapshot();
  });
});
