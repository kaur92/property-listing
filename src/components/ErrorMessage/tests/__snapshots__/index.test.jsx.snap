// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage /> tests renders correctly 1`] = `
<div
  className="error-page"
>
  <h1>
    Oops!!
  </h1>
  <h3>
    Something went wrong please try again.
  </h3>
</div>
`;
